test/test_transformers.py

```diff
@@ -1753,6 +1753,96 @@ def test_scaled_dot_product_fused_attention_vs_math_cpu(
             self.assertEqual(grad_k_actual, grad_k_ref, atol=atol, rtol=rtol)
             self.assertEqual(grad_v_actual, grad_v_ref, atol=atol, rtol=rtol)
 
+    @onlyCPU
+    @parametrize("fused_kernel", [SDPBackend.FLASH_ATTENTION])
+    @parametrize("dtype", [torch.float64, torch.float32, torch.bfloat16])
+    @parametrize("batch_size", [2, 12])
+    @parametrize("seq_len", [267, 1030])
+    @parametrize("n_head", [1, 3])
+    @parametrize("head_dim", [8, 16])
+    @parametrize("mask_dim", [2, 3, 4])
+    @parametrize("bool_mask", [0, 1])
+    @parametrize("train", [True, False])
+    def test_scaled_dot_product_fused_attention_mask_vs_math_cpu(
+        self,
+        device,
+        fused_kernel,
+        dtype,
+        batch_size,
+        seq_len,
+        n_head,
+        head_dim,
+        mask_dim,
+        bool_mask,
+        train,
+    ):
```

Comment:
There is `Tolerances` named tuple in this file that can be helpful for storing these

Reply:
Thanks and modified.